targets/TARGET_STM/TARGET_STM32F4/TARGET_STM32F439xI/TOOLCHAIN_ARM_STD/startup_stm32f439xx.S

```diff
@@ -0,0 +1,415 @@
+;******************** (C) COPYRIGHT 2017 STMicroelectronics ********************
+;* File Name          : startup_stm32f439xx.s
```

Comment:
I think this file addition is not related to this PR :-)

Reply:
No, it was not. I see that the directory `TOOLCHAIN_ARM_STD` was renamed `TOOLCHAIN_ARM` on master. I will fix that.
Thanks

Reply:
[Done](https://github.com/ARMmbed/mbed-os/pull/13566/commits/c0584531277fa2fa4a3293fe630c25cf5aab1628).